activejob/lib/active_job/exceptions.rb

```diff
@@ -116,7 +116,7 @@ def retry_job(options = {})
       def determine_delay(seconds_or_duration_or_algorithm)
         case seconds_or_duration_or_algorithm
         when :exponentially_longer
-          (executions**4) + 2
+          (executions**4) + 2 + Kernel.rand(30)
```

Comment:
Jittering 3 seconds by ~15 is quite a change whereas at higher values it isn't that much variation. We have something like this within our codebase:

```ruby
(((executions**4) + Kernel.rand(executions**4))/2) + 2
```

To compare the min-max values:

```
irb> (1..5).map { |i| [((i**4) + 2 + 0), ((i**4) + 2 + 30)] }
=> [[3, 33], [18, 48], [83, 113], [258, 288], [627, 657]]

irb> (1..5).map { |i| [((((i**4) + 0)/2) + 2), ((((i**4) + (i**4))/2) + 2)] }
=> [[2, 3], [10, 18], [42, 83], [130, 258], [314, 627]]
```

Reply:
Agreed. Low jitter at low execution count, higher at higher count.

Reply:
Great suggestion @gshutler - thanks for offering it up, I will fixup with the change.